src/ui/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,27 @@
+import { forwardRef } from 'react';
+
+import clsx, { ClassValue } from 'clsx';
+
+import { BoxProps, Box } from '../Box';
+import { base } from '../reset.css';
+import { sprinkles, Sprinkles } from '../sprinkles.css';
+
+import { skeletonCss } from './Skeleton.css';
+
+export interface SkeletonProps extends BoxProps, Pick<BoxProps, 'className' | 'height' | 'width' | 'borderRadius' | 'display' | 'flexDirection' | 'padding' | 'margin'>, Sprinkles {
+  className?: ClassValue;
+}
+
+const Skeleton = forwardRef(({ ...props }: SkeletonProps) => {
+  const styles = clsx(
+    sprinkles({
+      ...props,
+    }),
+    skeletonCss,
+    base
+  );
+
+  return <Box className={styles} />;
```

Comment:
```suggestion
const Skeleton = forwardRef(({ className,...props }: SkeletonProps) => {
  return <Box className={[skeletonCss, className]} {...props} />;
```

`Box`컴포넌트 내부에서 `sprinkles`를 처리하고 있어서, 이렇게만 작성해도 괜찮을 것 같습니다~!

Reply:
`Box` 컴포넌트에 배열로 넘겨준다는게 이런 의미였군요 🤔!!! 수정하겠습니다~!!

Reply:
수정했습니다~!!